docker-app/qfieldcloud/core/models.py

```diff
@@ -1213,6 +1213,15 @@ def delete(self, *args, **kwargs):
             storage.delete_project_thumbnail(self)
         super().delete(*args, **kwargs)
 
+    @property
+    def owner_can_create_job(self):
+
+        from qfieldcloud.core.permissions_utils import (
+            is_supported_regarding_owner_account,
+        )
+
+        return is_supported_regarding_owner_account(self)
+
     def check_can_be_created(self):
```

Comment:
IMO it finally is about `status of the project` where this check belongs. So the question is can there be Jobs triggered on the project. There are already NOTEs about refactoring project status - moving code here to make this easier. I removed the property form the delta to avoid abiguitiy.